tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_planning_server.h

```diff
@@ -0,0 +1,241 @@
+/**
+ * @file process_planning_server.h
+ * @brief A process planning server with a default set of process planners
+ *
+ * @author Levi Armstrong
+ * @date August 18, 2020
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2020, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef TESSERACT_PROCESS_MANAGERS_PROCESS_PLANNING_SERVER_H
+#define TESSERACT_PROCESS_MANAGERS_PROCESS_PLANNING_SERVER_H
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <memory>
+#include <string>
+#include <taskflow/taskflow.hpp>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract/tesseract.h>
+#include <tesseract_environment/core/types.h>
+#include <tesseract_command_language/core/instruction.h>
+#include <tesseract_command_language/null_instruction.h>
+
+#include <tesseract_motion_planners/core/profile_dictionary.h>
+#include <tesseract_motion_planners/descartes/profile/descartes_profile.h>
+#include <tesseract_motion_planners/trajopt/profile/trajopt_profile.h>
+#include <tesseract_motion_planners/ompl/profile/ompl_profile.h>
+#include <tesseract_motion_planners/descartes/profile/descartes_profile.h>
+#include <tesseract_motion_planners/simple/profile/simple_planner_profile.h>
+
+#include <tesseract_process_managers/process_environment_cache.h>
+#include <tesseract_process_managers/taskflow_generators/graph_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_global_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_only_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_only_global_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_dt_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_waad_taskflow.h>
+#include <tesseract_process_managers/taskflow_generators/raster_waad_dt_taskflow.h>
```

Comment:
These should be able to go away shouldn't they?

Reply:
Yea they can be moved to the cpp file.